.circleci/config.yml

```diff
@@ -14,10 +14,6 @@ jobs:
     working_directory: ~/plotly.js
     steps:
       - checkout
-      - restore_cache:
-          keys:
-            - v{{ .Environment.CIRCLE_CACHE_VERSION }}-deps-{{ .Branch }}-{{ checksum "package-lock.json" }}
```

Comment:
Notice here we use the checksum of "package-lock.json" but when we save the cache, we the checksum "package.json". Does two cache names will never match so this is why I removed this "dead" code.